.github/workflows/ci-workflow.yml

```diff
@@ -81,7 +81,8 @@ jobs:
           path: ./exchange_nanosp.elf
           if-no-files-found: error
 
-  jobs-e2e-tests-nano-S:
+  jobs_e2e_tests_nano_S:
```

Comment:
Yes, but it would need the python tests to also be available on NanoX and S+, because I don't think you can trigger a step after a specific matrix-step (?).
I do want to do followup branches on hardening the CI and adding tests.